tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
It's probably better practice to set this to true. You'll see there are lots of `any`s floating around and ideally we'd minimise them.

Reply:
Yea I left the defaults when converting the project with Next CLI. I've enabled this and have changed the various files which needed updating. Hopefully thats better now. Still feel that some things might need further updates.